### ISSUE TITLE
The Morgan Library and Museum (not supported)

### ISSUE THREAD

**Issue by koxivega** [lovasoa/dezoomify#101]
Doesn't work on this 
http://www.themorgan.org/collection/drawings/303869#overlay=node/3580/zoomify

**Comment by lovasoa** [comment]
Reporting an issue shouldn't prevent you from being polite. When reporting an issue, you are writing to human beings, not machines, and should start your message with something like *Hello*, and ending it with something like *thank you*.

**Comment by lovasoa** [comment]
And also, you should read the FAQ before reporting an issue. The URL you report works just fine, you just have to find the url of the metafile. How to do that is explained in the FAQ.

**Comment by lovasoa** [comment]
http://ophir.alwaysdata.net/dezoomify/dezoomify.html#http://www.themorgan.org/sites/default/files/zoomify/3580/4111/ImageProperties.xml

**Comment by koxivega** [comment]
I am really sorry, I didn't mean to be rude. And yes, I read the FAQ and tried myself the firefox method for finding the url through the XHR but the "ImageProperties.xml" didn't look significant to me, I don't know why.... probably I had to get back to the FAQ.
My fault I took for granted compliments and thanking you for the great unique tool you have offered to us all.
I am sincerely sorry, looks like I sounded like and asshole and that definitely I didn't mean to.
Kris
